mmaction/models/backbones/cuda_shift/setup.py

```diff
@@ -1,9 +1,24 @@
 import os
+import subprocess
 from setuptools import find_packages, setup
 
 import torch
 from torch.utils.cpp_extension import CUDAExtension
 
+
+def _find_cuda_home():
+    # guess rule 3 of torch.utils.cpp_extension
+    nvcc = subprocess.check_output(['which', 'nvcc']).decode().rstrip('\r\n')
```

Comment:
this function can be moved to util or mmcv

Reply:
Actually, this function is a simplified version of [_find_cuda_home](https://github.com/pytorch/pytorch/blob/master/torch/utils/cpp_extension.py#L27-L54) in PyTorch, which uses 3 Guess for `CUDA_HOME`. For the customized one, it directly sets directory path to `CUDA` which supports `nvcc` command as `CUDA_HOME`, which may be more reasonable.
The detailed description has been add as a comment.

Reply:
why change the cuda finding logic from pytorch? does it actually matters? or it was a temp workaround on the original authors machine?

Reply:
I have confirmed it with the author, he said they are the same in most cases. 
But I think the CUDA_HOME of nvcc is the one we expected.
Maybe I can try it with the PyTorch `_find_cuda_home`

Reply:
yeah if pytorch's cuda home works on our cluster, then it's better to remove this custom finding logic. if there are flaws in pytorch's logic, we can patch upstream